Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using System.Globalization;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public class Lesson3
+    {
+        public static void Demo()
+        {
+            const int maxEntries = 2;
+
+            for (int i = 1; i <= maxEntries; i++)
+            {
+                Console.Clear();
+
```

Comment:
A blank line in between every line of code is not needed.

Separate unrelated code with a blank line and if all code is unrelated- such separation is not needed.